src/Codeception/Module/AMQP.php

```diff
@@ -305,6 +305,55 @@ public function seeMessageInQueueContainsText($queue, $text)
         $this->assertContains($text, $msg->body);
     }
 
+    /**
+     * Count messages in queue.
+     *
+     * @param string $queue
+     *
+     * @return int
+     */
+    public function countMessageInQueue($queue)
```

Comment:
I think that this method shouldn't be a part of public API.
Leaving it public but prepending undescore to the name (`_countMessageInQueue`) would hide it in tests but allow modules ands helpers to use it.

Reply:
I've made the changes